docs/maintenance/upgrade_alert.md

```diff
@@ -23,8 +23,8 @@ With these caveats clarified, let's define an upgrade alert. The instructions we
 
 Step 1. Add a PostgreSQL data source with the following credentials:
 
-* `telemetry_testnet` for testnet: https://github.com/near/near-explorer/blob/master/backend/config/env-indexer-testnet
-* `telemetry_mainnet` for mainnet: https://github.com/near/near-explorer/blob/master/backend/config/env-indexer-mainnet
+* `telemetry_testnet` for testnet: https://github.com/near/near-explorer/blob/master/backend/config
+* `telemetry_mainnet` for mainnet: https://github.com/near/near-explorer/blob/master/backend/config
```

Comment:
@janewang I feel it is better to refer users to https://github.com/near/near-indexer-for-explorer/#shared-public-access. If you find URI format hard to reason about, consider submitting a PR to that README file with the clarifications.

Either way, the link you use here won't help the readers as there is nothing they can use there. Those files that were originally referenced were moved to:

* https://github.com/near/near-explorer/blob/master/mainnet.env#L9-L12
* https://github.com/near/near-explorer/blob/master/testnet.env#L9-L12